src/transformers/pipelines/__init__.py

```diff
@@ -859,6 +859,8 @@ def pipeline(
                 'You cannot use both `pipeline(... torch_dtype=..., model_kwargs={"torch_dtype":...})` as those'
                 " arguments might conflict, use only one.)"
             )
+        if torch_dtype is str:
+            torch_dtype = getattr(torch_dtype, torch_dtype)
```

Comment:
I don't think this is compatible with passing in `torch_dtype="auto"`, which the docstring says it supports

Reply:
sorry for the late reply, taking a look now